### ISSUE TITLE
Many users don't recognize that the bottom menu is scrollable

### ISSUE THREAD

**Issue by fofwisdom** [ronilaukkarinen/mastodon-bird-ui#26]
What if we improved the bottom menu on mobile in this way?
- https://social.silicon.moe/@perillamint/110291024326379366
![c4e1244df7a65224](https://user-images.githubusercontent.com/205521/235396523-8a384ef4-e0ee-45c7-a596-ec4016573f27.jpg)
Thanks as always for sharing good design with everyone.

**Comment by perillamint** [comment]
Here is my patch with some flaw: Does not work well while handling logged out user

```
diff --git a/style.css b/style.css
index 3c567db..93df30a 100644
--- a/style.css
+++ b/style.css
@@ -2947,7 +2947,7 @@ body.embed .button.logo-button:hover,
   /* Make the column link 1/4 of width of the screen */
   .layout-single-column .navigation-panel .navigation-panel__legal,
   .layout-single-column .navigation-panel .column-link {
-    flex: 0 0 calc(100vw / 4);
+    flex: 0 0 calc(100vw / 4.5);
     padding: 0;
   }
```

**Comment by ronilaukkarinen** [comment]
I really like they're equal. I understand the scrollability is not obvious, but there's really not much options here. Have to think about this further. For the time being you can always edit your own CSS.

**Comment by Roboron3042** [comment]
Just want to comment that when presented with this issue I also reached the same conclusion as [fofwisdom](https://github.com/fofwisdom).

**Comment by ronilaukkarinen** [comment]
Hmm, actually kinda good idea from @mstdn:

> Maybe stick with 3/4 buttons and a "more" or something like that? thought_balloon

It could be possible to hide the rest of the links behind three dots or similar "more" icon, and open a context menu where the rest of them will be after clicking that last icon. This way no horizontal scroll would be needed.

It needs a tremendous amount of styles under `:focus` and `:hover` selectors for this to work. I'll think about it when I have more time.

**Comment by itbeard** [comment]
On our server (VKL.world) we changed ordering and set 6 icons in the row:
![image](https://github.com/ronilaukkarinen/mastodon-bird-ui/assets/45439635/0a8e4854-e0c3-497a-90ed-f0e093344735)

Code:
```
@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .navigation-panel__legal, .layout-single-column .navigation-panel .column-link {
  flex: 0 0 calc(100vw / 6);
  padding: 0;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(2) {
  order: 1;
}}

@media screen and (max-width: 889px){
.layout-single-column .navigation-panel .column-link:nth-child(5) {
  order: 2;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(6) {
  order: 3;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(4) {
  order: 4;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(3) {
  order: 5;
}}

@media screen and (max-width: 889px){
.layout-single-column .navigation-panel .column-link:nth-child(7) {
  order: 6;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(8) {
  order: 7;
}}

@media screen and (max-width: 889px) {
.layout-single-column .navigation-panel .column-link:nth-child(9) {
  order: 8;
}}

@media screen and (max-width: 889px){
.layout-single-column .navigation-panel .column-link:nth-child(10) {
  order: 9;
}}
```

**Comment by fofwisdom** [comment]
> Code:

When I erased the existing one and applied /* mobile */ where it was attached, the mobile sidebar stretched 9 spaces to the left.

**Comment by ronilaukkarinen** [comment]
For big thumbers and smaller phone models that's too much. But feel free to modify! :+1:

**Comment by derLaokrat** [comment]
![scroll indicator](https://github.com/ronilaukkarinen/mastodon-bird-ui/assets/139688318/07bdf562-d499-4c21-b7d5-53ae72db3ae5)
How about a simple scroll indicator like this? with a tiny glow around the purple part?

**Comment by ronilaukkarinen** [comment]
Hmm. For me personally that is a bit of a distraction. Also not aesthetically pleasing to have it always there. Especially on mobile hate scrollbars, they are not often needed (you can't drag them) and taking visually too much space.

I have kinda come to a conclusion that it's not the end of the world if some don't realize its swiping abilities. We can't fit all of them there anyway.

In fact for these reasons I'm going to close this for now. I hope you understand.

**Comment by wandy-dev** [comment]
I was going to open a separate issue, but I figured this was the most relevant. Perhaps options for selecting which icons are up front can be helpful for some. I don't really mind if my users scroll, but I have a small instance with no relays so the the explore tab is empty. I'd love to move Home, Live feeds, Private mentions, and Preferences to the front.

Once this is done that'll cover most of my users needs and I can just let people know they can scroll the tabs bar for my "super users"

**Comment by ronilaukkarinen** [comment]
> Perhaps options for selecting which icons are up front can be helpful for some.

You can edit the order in the CSS code to your likings. No options can be done with styles as this is not an app or any core modification and not in any way related to functional code.

**Comment by wandy-dev** [comment]
As a back end developer, do you think you can point me in the right direction for this? I'm not sure how straight forward, or complicated this might be.

**Comment by Shanesan** [comment]
I made the following tweak (you can add this to the very top of the CSS to try it out). I don't propose this to be the fix, but it definitely helps tell someone that there is more, subtly.

<img width="478" alt="image" src="https://github.com/ronilaukkarinen/mastodon-bird-ui/assets/22644226/a22a4204-2a97-4b68-80ef-c05c13942a5a">

```
/* nav bar */
.layout-single-column .columns-area__panels__pane--navigational .navigation-panel {
	background: linear-gradient(90deg, #08171e00, #08171e00, #08171e00, #08171e00, #08171e00, #08171e00, #08171e00, #1e3b51) repeat fixed top left;
}
```
